source/sync/configure/enable-sync.txt

```diff
@@ -132,6 +145,13 @@ Enable Partition-Based Sync
      
    .. tab::
       :tabid: cli
+
+      This procedure assumes you have :ref:`created an App Services App 
+      <create-app-cli>` and have linked a data source.
```

Comment:
Might want to link to [link a data source](https://www.mongodb.com/docs/atlas/app-services/mongodb/#link-a-data-source) here